Sources/SpeziViews/ViewModifier/SkeletonLoading.swift

```diff
@@ -0,0 +1,67 @@
+//
+// This source file is part of the Stanford Spezi open-source project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University and the project authors (see CONTRIBUTORS.md)
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SwiftUI
+
+
+public struct SkeletonLoading<SkeletonCell: View>: View {
+    var cellCount: Int
+    var shimmerRepeatInterval: Double
+    let skeletonCell: () -> SkeletonCell
+
+    
+    public var body: some View {
+        VStack(spacing: 16) {
+            ForEach(0..<cellCount, id: \.self) { _ in
```

Comment:
Also we might want to make sure that we do not allow `cellCount` values below 0 and automatically correct them to zero. Otherwise this ends up crashing the application in the range expression above.

Reply:
Good point, should be checked in the initialized when defining the count.